src/freenet/clients/http/geoip/IPConverter.java

```diff
@@ -1,15 +1,22 @@
 package freenet.clients.http.geoip;
 
+import java.io.BufferedReader;
 import java.io.File;
-import java.io.FileNotFoundException;
 import java.io.IOException;
-import java.io.RandomAccessFile;
+import java.io.InputStream;
+import java.io.InputStreamReader;
 import java.lang.ref.SoftReference;
 import java.lang.ref.WeakReference;
+import java.nio.file.NoSuchFileException;
+import java.util.ArrayList;
 import java.util.Arrays;
+import java.util.Collections;
 import java.util.HashMap;
 import java.util.LinkedHashMap;
+import java.util.List;
 import java.util.Map;
+import java.util.zip.ZipException;
```

Comment:
These are needed despite no other changes to this file?

Reply:
> These are needed despite no other changes to this file?

It appears Arne forgot to include the other changes in IPConverter, which you can see are explained in a way that it is already done.  I figure he intends to come back to paste the remaining changes later.

Reply:
@ArneBab Are these changes incomplete? Is this a draft?